### ISSUE TITLE
Offline songs to stream in playlist mode not downloads

### ISSUE THREAD

**Issue by Anonymous18-101** [jhelumcorp/gyawun#182]
Offline songs to stream in playlist mode not downloads so here is the simplest explanation lets say you downloaded song name A and you have it in your downloads with names like 1.2.3.4.5..6 something like that but you found a song that is new named b and you downloaded and it's in downloads so the A,b are english 1.2.3.4.5..6 are japanese but song b is after the japanese order in downloads. If i want to listen to a japanese playlist offline i can simply open playlist and listen to offline playlists not online or downloads where everything is jumbled with songs which are not in order 
![Screenshot_2024-11-21-21-09-05-974_com.jhelum.gyawun.jpg](https://github.com/user-attachments/assets/6a0b9c2a-8d14-45cb-9bec-36fe06326723)

 So basically make the app to stream offline playlists not downloads where i have to roam the downloads which you can understand the picture i tagged and the downloads is laggy i don't know I even updated the recent one you released so please make this happen it will help me :)
